vm.worker.js

```diff
@@ -109,6 +109,10 @@
 	    this.vmWorker.postMessage({method: 'stopAll'});
 	};
 
+	VirtualMachine.prototype.animationFrame = function () {
```

Comment:
hmmm... yeah, it's just appearing for the first time in the compiled version. that's already in the code.